.github/workflows/eaa_kbc.yml

```diff
@@ -14,45 +16,35 @@ jobs:
   eaa_kbc_ci:
     if: github.event_name == 'pull_request'
     name: Check
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
         rust:
           - stable
+
+    container: runetest/compilation-testing:ubuntu18.04
```

Comment:
Is there a 20.04 version of this container? Why are we using the `rune` container here?

Reply:
`rune` is a historical name. Because the rats-tls that EAA KBC relies on was originally a subproject under the `rune` project, its compilation test container is placed under the runetest namespace. I learned that 20.04 version of this container is under development.

Since this is just a compilation test, we can temporarily use the old version of the container to compile rats-tls.